tools/export.py

```diff
@@ -232,15 +240,19 @@ def main(args):
         for k, v in nncf_part.items():
             cfg[k] = v
 
+    is_alt_ssd_export = getattr(args, 'alt_ssd_export', False)
+    if is_alt_ssd_export:
+        patch_model_for_alt_ssd_export(model)
```

Comment:
Why have you moved this out of `export_to_onnx` function? The flow becomes somewhat inconsistent now, because extra symbolics registration and patching for default mode happens there, while the same thing for alternative SSD mode is here. Also I'd not put this logic inside the BEGIN-END block for NNCF (yeah, I know that those are only comments, but it confuses).

Reply:
Agree, moved above NNCF block